testsuite/integration-tests-spring/inmodule/src/test/resources/arquillian.xml

```diff
@@ -7,13 +7,14 @@
     <container qualifier="jbossas-managed" default="true">
         <configuration>
             <property name="jbossHome">${jboss.home}</property>
-            <property name="jbossArguments">${securityManagerArg}</property>
             <property name="serverConfig">${jboss.server.config.file.name:standalone.xml}</property>
             <!--<property name="javaVmArguments">-Xmx512m -XX:MaxPermSize=128m
                 -Xrunjdwp:transport=dt_socket,address=8787,server=y,suspend=y
             </property>-->
-            <property name="javaVmArguments">${debugJvmArgs} -server -Xms96m -Xmx1024m -Djboss.bind.address=${node} -Djboss.bind.address.management=${node} -Dnode=${node} -Dipv6=${ipv6} ${additionalJvmArgs} ${ipv6ArquillianSettings} ${jacoco.agent}</property>
-            <property name="managementAddress">${node}</property>
+<!--            <property name="javaVmArguments">${debugJvmArgs} -server -Xms96m -Xmx1024m -Djboss.bind.address=${node} -Djboss.bind.address.management=${node} -Dnode=${node} -Dipv6=${ipv6} ${additionalJvmArgs} ${ipv6ArquillianSettings} ${jacoco.agent}</property>-->
+            <property name="javaVmArguments"></property>
```

Comment:
Not a huge deal, but it looks like the `debugJvmArgs` are defined in the POM so it should work here to. Up to you though :)

Reply:
Thanks for checking James! I'll do some cleanup work and finalize this PR :D